Client/src/Components/Search/Search.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
+//import { Container,Form, FormControl, FormGroup, Table } from '@material-ui/core';
+import {Form,  Button,Col, Container, Table} from 'react-bootstrap/'
+
+ function Search(){
+    return (    
+    <Container>
+        <div>
+          <h1> SEARCH </h1>
+        </div>
+
+        <Form>
+            <Form.Row>
+                <Form.Group as={Col} controlId="formGridOxidizer">
+                    <Form.Control defaultValue="Oxidizer" />
+                </Form.Group>
+
+                <Form.Group as={Col} controlId="formGridYear">
+                    <Form.Control defaultValue="YEAR" />
+                </Form.Group>
+
+                <Form.Group as={Col} controlId="formGridOutputFormat">
+                {/* <Form.Label>State</Form.Label> */}
+                    <Form.Control as="select" defaultValue="NONE">
+                        <option>Output Formats</option>
+                        <option>DEEZ NUTS</option>
+                    </Form.Control>
+                </Form.Group>
+
```

Comment:
This component is missing all API calls to backend to fill the table. If you need to see an example of how this is done you check check the fileUpload branch.

Im not sure we can complete the PR without it since if we complete another dev will be required to come back and modify the code for the story